src/components/TextInput/BaseTextInput.js

```diff
@@ -291,8 +295,7 @@ class BaseTextInput extends Component {
                                     {this.props.secureTextEntry && (
                                         <Checkbox
                                             style={styles.secureInputShowPasswordButton}
-                                            onPress={this.togglePasswordVisibility}
-                                            onMouseDown={e => e.preventDefault()}
+                                            onMouseDown={this.togglePasswordVisibility}
```

Comment:
This can't be working since there is no `onPress` handler

https://user-images.githubusercontent.com/9059945/204882403-1fd5823b-0ea4-471e-ad8d-14482c803df8.mov

It just does nothing on native and mWeb
Please **test all the platforms** every time you open a new PR

Reply:
In the handler for native(checkbox.native.js), it automatically adds the onPress if it is not present. Or at least it used to....